coredump/coredump

```diff
@@ -48,6 +48,5 @@ def main():
         print('Exiting')
         sys.exit(1)
 
-
 if __name__ == '__main__':
```

Comment:
Expected 2 blank lines after end of function or class (E305)
https://www.flake8rules.com/rules/E305.html